server/container/core/pom.xml

```diff
@@ -141,7 +141,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <reuseForks>true</reuseForks>
+                    <reuseForks>false</reuseForks>
```

Comment:
is it related?

Reply:
Yes because the environment variable is checked once and then cached, so, reusing the fork mean that setting the environment variable have no effect.